### ISSUE TITLE
[Section] Testimonial Cards

### ISSUE THREAD

**Issue by Cijin** [nbanker/tapuo#6]
* Non existent at the moment, needs to be created

**Comment by nbanker** [comment]
Short

" [Kareem Felix] Friendly team, They got everything done in a short time"


"[Alma Mccarthy] I had a problem and could not find the issue, Tapuo help me sort it out in no time "


"[Samuel Dodson]  reliable, fast and thorough, I will be back"


"[Tiah Busby]  They helped me realize an idea I had for a long time and did not know how to get it going, business is booming now, Thanks Guys "




Long

"- Herman Keeling-  I have a home travel based business and wanted to take it online. I did not know how to set it up, my designs were awful. The Tapuo Team turned it around. They knew how to go about it and had a lot of great ideas. After they were done I had a well visited site and lots of Travellers "



"- Lia Feeney-  I had my shop set up and ready to go. Then nothing happend. Nearly no visitors and the few that came left after only a short time in my shop. I contacted Tapuo and they gave me lots of free advice. I decided to let them take the wheel for some time and when they returned it my shop had transformed"



"- Abel Lawson- My was running well, but I had to spend 6 -7 hours ever day on only one task. Data entry, I had little to now time to take care of clients, answer their emails or improve my business and take it to the next level. And no time for family. That had to change. The Tapuo Team streamlined it for me, I have a lot of time now to spend with my daughter and wife "



"- Rian Kirby- I lost my job in these difficult times. I heard that there is opportunity to become an online business from home. Someone recommended these guys from Tapuo to me. I had a chat and some weeks later I decided to get started and let them help me. My regrets are that I waited so long, Thanks Tapuo Love you guys "
